### ISSUE TITLE
Error displaying the character when changing weapons/armor

### ISSUE THREAD

**Issue by M1sh2** [alexbatalov/fallout1-ce#35]
It seems that the error is present in both versions: f1 ce, f2 ce, after a long game, the display of the character in the inventory and in the game itself begins to differ.
![incorrect2](https://user-images.githubusercontent.com/59886778/219563584-721bbdce-7e75-4e5e-b135-e42af6bc6c0a.jpg)
![incorrect_2_1](https://user-images.githubusercontent.com/59886778/219563590-031c96d5-c8d3-4970-ac85-e7b286a1cd80.jpg)
![incorrect2_2](https://user-images.githubusercontent.com/59886778/219563598-f0c66a60-8e97-47fb-a865-3acc72ac47a6.jpg)

**Comment by alexbatalov** [comment]
I'm not sure what to look for? You mean you changed robe to combat armor, but it was not reflected in the game?

**Comment by M1sh2** [comment]
> I'm not sure what to look for? You mean you changed robe to combat armor, but it was not reflected in the game?

Yes, this also happens with weapons, when the old weapon is displayed, but you can not use it, of course. I have to enter the inventory again to update the character.
